### ISSUE TITLE
Show4D: add vmin/vmax for absolute intensity range

### ISSUE THREAD

**Issue by bobleesj** [bobleesj/quantem.widget#81]
## Problem

Show4D has no way to set absolute intensity bounds. When comparing signal frames across different nav positions, each frame auto-ranges to its own min/max, making visual comparison misleading.

## Proposed Solution

Add `vmin`/`vmax` traits (same pattern as Show2D/Show3D/Show3DVolume). When set, both nav and signal panels use absolute bounds instead of per-frame auto-ranging.

```python
Show4D(data, vmin=0, vmax=1000)
```

## Files

- `src/quantem/widget/show4d.py` — add traits, update `__init__`, `_normalize_frame`, `state_dict`, `summary`
- `js/show4d/index.tsx` — read traits, update rendering effects
- `tests/test_widget_show4d.py` — constructor, state roundtrip, normalize, summary tests

**Comment by henryhng** [comment]
Could this use the same implementation as Show2D #76 ? Needs nav_vmin/vmax + sig_vmin/vmax for dual panel... similarly
for #82 : Needs dp_vmin/vmax + vi_vmin/vmax, plus handle scale_mode (log/power) transform.
for #83 & #84 : same pattern as Show2D

edit: nevermind, didn't see the proposed solution section at first!
